src/js/config/Config.template.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
     },
     introduction: {
       enabled: false
+    },
+    'remove-oauth': {
+      enabled: false
```

Comment:
You're right, this is an oversight.
This file should export an empty object until we have installer plugins. The rest of this file should be removed and move to the private plugins repo.